scripts/common/common.sh

```diff
@@ -1,6 +1,3 @@
-
-STORAGE_PROVISIONER=rook
```

Comment:
Why is this going away?

Reply:
`STORAGE_PROVISIONER` is unused anywhere in the code